mantle/cmd/ore/do/create-image.go

```diff
@@ -108,14 +108,26 @@ func createSnapshot() error {
 	if err != nil {
 		return err
 	}
-	defer API.DeleteKey(ctx, keyID)
+	defer func() {
+		err := API.DeleteKey(ctx, keyID)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(1)
```

Comment:
I think even if we hit an error we still want to do the other cleanups, that's why errors were ignored before.
So let's drop the `os.Exit()`s?

Reply:
Make sense. Thanks for pointing out.